docs/source/entities-advanced.mdx

```diff
@@ -270,9 +221,58 @@ We're done! `Bill` now originates in a new subgraph, and it was resolvable durin
 
 </ExpansionPanel>
 
+<ExpansionPanel title="With Rover CLI composition">
+
+1. In the Billing subgraph, define the `Bill` entity exactly as it's defined in the Payments subgraph:
+
+    ```graphql title="Billing subgraph"
+    type Bill @key(fields: "id") {
+      id: ID!
+      amount: Int!
+    }
+    ```
+
+3. In the Billing subgraph, define resolvers for each field of `Bill` that are currently resolved in the Payments subgraph. These resolvers should behave identically to the resolvers in the Payments subgraph.
```

Comment:
Should be a 2 here but also it look like MD doesn't care and it's correct on the docs site as-is anyway. Don't care if you bother to change it or not.

Reply:
Unacceptable!!! I've fixed this _glaring_ bug